i18n/management/commands/publish_i18n.py

```diff
@@ -59,29 +58,35 @@ def has_changes(self, language_code):
         return len(self.changes.get(language_code, [])) > 0
 
     def publish_objects_in_language(self, objects, language_code):
-        translation.activate(language_code)
-
         # By default, the spawned processes will attempt to reuse the existing database connections,
         # which will cause conflicts. To prevent this, we explicitly close all database connections
         # immediately before spawning the processes, which will force them to each open their own
         # connection.
         db.connections.close_all()
 
+        translation.activate(language_code)
+
+        pool = multiprocessing.Pool(multiprocessing.cpu_count())
```

Comment:
what's the advantage of creating a separate `pool` variable vs `self.pool` ?

Reply:
I made this change to guarantee the child process inherited the parent process state after `translation.activate(language_code)` is called. Its probably true that I could revert `pool` back to a instance variable, but being an instance variable seems unnecessary anyways because no other class methods are using `pool`.